server/models/handlers/input_schedule.js

```diff
@@ -0,0 +1,15 @@
+// Create stub to input weekly worker schedule into database 
+// Appointments occur at intervals of 30 minutes for each session for a 24 hour period Monday to Friday 
+// Service worker must input their avability at each corresponding school for the week. This recurres for 4 weeks
```

Comment:
Code comments should be at a higher abstraction than the code. Thus, it might be better to omit line 4 or reword it to be more human readable.